docs/_articles/en/einstein/einstein-overview.md

```diff
@@ -9,16 +9,15 @@ Einstein for Developers is an AI-powered developer tool that's available as an e
 
 Einstein for Developers assists you throughout the Salesforce development process with expertise learned from anonymized code patterns. Our suite of AI-powered developer tools increases productivity and provides helpful assistance for complex coding tasks. We enforce development best practices with code generation and our suite of recommended static analysis and security scanning tools. With boilerplate code generation as its foundation, AI-assisted tooling also makes it easier for new developers to onboard to the Salesforce Platform.
 
-
 **Important**: This feature is a Beta Service. A customer may opt to try such Beta Service in its sole discretion. Any use of the Beta Service is subject to the applicable Beta Services Terms provided at [Agreements and Terms](https://www.salesforce.com/company/legal/agreements/).
 
 ## Current Capabilities
 
 Einstein for Developers generates Apex code from natural language prompts and automatically suggests code completions for you as you type. When enabled along with IntelliSense, this feature makes Salesforce development tooling in Visual Studio Code even richer. Familiarity with Visual Studio Code is assumed.
 
-* Enter natural language instructions in a sidebar, so you can work with your editor and the tool side by side, without any interruptions to your workflow. Or use the VS Code Command Palette to enter a prompt describing what you'd like to build and then generate code suggestions within your editor.
-* Use inline autocompletion to automatically receive suggestions as you write Apex and LWC code.
-* Generate Apex unit tests to quickly accomplish required code coverage to get your Apex code ready for deployment. 
+- Enter natural language instructions in a sidebar, so you can work with your editor and the tool side by side, without any interruptions to your workflow. Or use the VS Code Command Palette to enter a prompt describing what you'd like to build and then generate code suggestions within your editor.
```

Comment:
Limited LWC code sounds a bit funky to me here...can we say something like `...as you write Apex and LWC (JS & CSS only)` I don't love that either....

Can we add somewhere explicitly that we don't have HTML support right now but we'll be releasing that shortly?

Reply:
^^ Ideally in the inline doc section for sure